static/js/stat_var_hierarchy/stat_var_search.tsx

```diff
@@ -107,27 +114,31 @@ export class StatVarHierarchySearch extends React.Component<
         </div>
         {renderResults && (
           <div className="statvar-hierarchy-search-results" tabIndex={-1}>
+            {showResultCount && (
+              <div className="result-count-message">
+                Matches {this.state.svgResults.length} groups and {numStatVars}{" "}
```

Comment:
nit: pluralize `groups` and `variables`?

Reply:
I think they're already pluralized?

Reply:
ohh, right, will make that fix